templates/test/overview.html.ep

```diff
@@ -89,6 +89,12 @@
                     <input type="text" class="form-control" name="arch" placeholder="any" id="filter-arch">
                 </div>
                 <div class="form-group">
+                    <strong>Failed modules</strong>
+                    <input name="failed_modules" type="text" id="filter-failed_modules" placeholder="shutdown,install_and_reboot">
+                    <%= help_popover('Help for the <em>Failed Modules</em> filter' => '
+                        <p>Shows only the jobs that failed the specified modules</p>') %>
```

Comment:
s/jobs that failed the specified modules/jobs that failed **in** the specified modules/